### PR TITLE
use endpoint data for generator efficency

### PR DESCRIPTION
**Pull request by wazkayd**

### CODE REVIEW

File: src/components/pieCharts/ScoreCardGenEfficiencyDoughnut.js [hunk 1]
```diff
@@ -10,8 +10,10 @@ const ScoreCardGenEfficiencyDoughnut = ({ data, uiSettings, peakData }) => {
 
   const colorAndMessage = getGeneratorSizeMessage(usage);
 
-  const percentageUsage = (peakData?.peak / gen_size) * 100;
-  const roundedUsage = percentageUsage.toFixed(2);
+  // const percentageUsage = (peakData?.peak / gen_size) * 100;
+  const percentageUsage = usage;
+  // const roundedUsage = percentageUsage.toFixed(2);
+  const roundedUsage = percentageUsage
```
Comment (idmega2000) [inline src/components/pieCharts/ScoreCardGenEfficiencyDoughnut.js:16]: weldone @wazkayd , However I think there is no need declaring another variable hold usage i.e  const percentageUsage = usage; you can pass usage directly where it is needed

**Review by idmega2000 — CHANGES_REQUESTED** [review]
Check review and fix

File: src/components/pieCharts/ScoreCardGenEfficiencyDoughnut.js (hunk 1 above)
Comment (wazkayd) [reply src/components/pieCharts/ScoreCardGenEfficiencyDoughnut.js]: its now resolved
you may merge please
thanks

**Review by idmega2000 — APPROVED** [review]
LGTM

**Pull request merged**